pkgs/development/libraries/xmlsec/default.nix

```
@@ -16,6 +16,7 @@ stdenv.mkDerivation {
 
   patches = [
     ./lt_dladdsearchdir.patch
+    ./remove_bsd_base64_decode_flag.patch
   ];
```

I think this patch is darwin-especific. Would you mind updating this bit?
```suggestion
  patches = [
    ./lt_dladdsearchdir.patch
  ] ++ stdenv.lib.optionals stdenv.isDarwin [ ./remove_bsd_base64_decode_flag.patch ];
```
besides that, lgtm.